react/tic-tac-toe/src/redux/users/actions.js

```diff
@@ -0,0 +1,28 @@
+import { push } from 'connected-react-router';
+
+import login from '../../services/UsersService';
+import api from '../../config/api';
+
+
+export const actions = {
+  LOGIN: '@@LOGIN/LOGIN',
+  LOGIN_FAILURE: '@@LOGIN/LOGIN_FAILURE',
+  LOGIN_SUCCESS: '@@LOGIN/LOGIN_SUCESS'
+};
+
+const actionsCreators = {
+  login: data => async dispatch => {
+    dispatch({ type: actions.LOGIN });
+    const response = await login(data);
+    if (response.ok) {
+      dispatch({ type: actions.LOGIN_SUCCESS });
+      localStorage.setItem('token', response.data.token);
+      api.setHeader('token', response.data.token);
```

Comment:
This is okay, but use the store to hold the token too. So, when you need to access it you keep data flow consistency. Then, in your default state you can something like this: 

```js
const initialState = {
  isLogged: localStorage.getItem('token');
};
```